core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -301,7 +304,10 @@ public int read(byte[] b, int off, int len) throws IOException {
       return -1;
     }
     int toRead = Math.min(len, mCurrentChunk.readableBytes());
-    mCurrentChunk.readBytes(b, off, toRead);
+    try (Timer.Context ctx = MetricsSystem
```

Comment:
@LuQQiu , I am a bit concerned about the overhead introduced here.
Can you add a flag to turn on and off whether we are tracking these operations by timers?